DependencyInjection/FOSRestExtension.php

```diff
@@ -37,6 +40,29 @@ public function prepend(ContainerBuilder $container)
         if ($config['view']['view_response_listener']['enabled']) {
             $container->prependExtensionConfig('sensio_framework_extra', ['view' => ['annotations' => false]]);
         }
+
+        $bundles = $container->getParameter('kernel.bundles');
+        if (isset($bundles['WebProfilerBundle'])) {
+            $profilerConfigs = $container->getExtensionConfig('web_profiler');
```

Comment:
I am still not convinced that this is the best solution here. We would now depend on assuming how the web profiler config is processed internall which makes our code very fragile to break in case they make some changes there.